common/shlibs

```diff
@@ -324,7 +324,7 @@ libglademm-2.4.so.1 libglademm-2.6.7_1
 libthunarx-3.so.0 Thunar-1.8.1_1
 libexif.so.12 libexif-0.6.17_1
 liboil-0.3.so.0 liboil-0.3.16_1
-libopenmpt.so.0 libopenmpt-0.4.11_1
+libopenmpt.so.0 libopenmpt-0.5.3_1
```

Comment:
No need to change this version unless there is a known ABI change. The purpose of `common/shlibs` is to record the minimum version number that provides a specific soname.

Reply:
@ahesford Understood, reverted.